app/models/form/sales/pages/person1_age.rb

```diff
@@ -0,0 +1,22 @@
+class Form::Sales::Pages::Person1Age < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = "person_1_age"
+    @header = ""
+    @description = ""
+    @subsection = subsection
+    @depends_on = [
```

Comment:
Not a comment on this so much as a general thought, it might be good to refactor our deps to support conditions, now that we're in a fully-fledged programming language rather than JSON.

Reply:
Completely agree, I hate this too :D